src/Compilers/VisualBasic/Test/Semantic/Semantics/LambdaTests.vb

```diff
@@ -2097,5 +2097,70 @@ End Class
 ]]>)
         End Sub
 
+        <WorkItem(1207506, "DevDiv"), WorkItem(4899, "https://github.com/dotnet/roslyn/issues/4899")>
+        <Fact()>
+        Public Sub InitClosureInsideABlockInAConstructor()
+
+            Dim compilationDef =
+<compilation>
+    <file name="a.vb">
+Imports System
+
+Module Module1
+
+    Sub Main()
+        Dim f As New FpB(100, 100)
+        System.Console.WriteLine(f.FPixels.Length)
+    End Sub
+
+End Module
+
+Public Class FpB
+    Public Property FPixels() As FloatPointF(,)
+        Get
+            System.Console.WriteLine("In getter")
+            Return m_FPixels
+        End Get
+        Set(value As FloatPointF(,))
+            m_FPixels = value
+        End Set
+    End Property
+    Private m_FPixels As FloatPointF(,)
+
+
+    Public Sub New(width As Integer, height As Integer)
```

Comment:
It is very likely we have something like this. The existing code ensures that we are not going to attempt to capture Me until constructor initializer is called, regardless of its arguments.